### PR TITLE
Branch field does not load -- hold for 2.6

### PR DESCRIPTION
**Pull request by jeswanke**
https://github.com/stolostron/backlog/issues/26414

Signed-off-by: John Swanke <jswanke@redhat.com>

### CODE REVIEW

File: frontend/src/routes/Home/Search/components/Modals/DeleteResourceModal.test.tsx [hunk 1]
```diff
@@ -217,17 +217,22 @@ describe('DeleteResourceModal', () => {
             expect(submitButton).toBeTruthy()
             userEvent.click(submitButton)
 
-            // Wait for delete resource requesets to finish
-            await waitForNocks([deleteResourceNock])
+            // // Wait for delete resource requesets to finish
```
Comment (chenz4027) [inline frontend/src/routes/Home/Search/components/Modals/DeleteResourceModal.test.tsx:220]: @jeswanke Do you plan to remove it?

**Review by chenz4027 — APPROVED** [review]
/lgtm

File: frontend/src/routes/Home/Search/components/Modals/DeleteResourceModal.test.tsx (hunk 1 above)
Comment (jeswanke) [reply frontend/src/routes/Home/Search/components/Modals/DeleteResourceModal.test.tsx:220]: @chenz4027 hey Maggie--good question--I just moved it to the bottom of the test---this test has been randomly ending before this nock has been finished--not sure what funkiness is going on with where that nock wait was located

Comment (chenz4027) [reply frontend/src/routes/Home/Search/components/Modals/DeleteResourceModal.test.tsx:220]: Ah I see, thanks John for the explanation :)

**Pull request merged**